### PR TITLE
Completed homework program given on 18 Jan 2020

### PR DESCRIPTION
**Pull request by ChaitanyaKDec19**

### CODE REVIEW

File: src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * 1. Capital Letters
+ * 2. Digit - Odd number sum
+ * 3. Special Characters
+ * 4. Even Number sum
+ * 5. Small Characters
+ */
+
+public class HomeWork {
+
+	static void displayString(String string) {
```
Comment (mkanani13) [inline src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java]: It is always advisable not to have same variable as datatype. You should have variable name may be as inputString.

File: src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+				capitalLetters = capitalLetters + string.charAt(i);
+
+			} else if (Character.isLowerCase(string.charAt(i))) {
+
+				smallLetters = smallLetters + string.charAt(i);
+
+			} else if (Character.isDigit(string.charAt(i))) {
+
+				ch = ch + string.charAt(i);
```
Comment (mkanani13) [inline src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java]: I think we can remove this line.

Comment (ChaitanyaKDec19) [reply src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java]: I believe you want me to remove line 39

File: src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java (hunk 1 above)
Comment (ChaitanyaKDec19) [reply src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java]: Sure I will keep this in mind going forward 👍👍

**Pull request merged**